dash/_utils.py

```diff
@@ -149,6 +149,12 @@ def _concat(x):
             _id += f"@{hashed_inputs}"
         return _id
 
+    if no_output:
+        # No output will hash the inputs.
+        return hashlib.sha256(
+            ".".join(str(x) for x in inputs).encode("utf-8")
+        ).hexdigest()
```

Comment:
Any way to avoid duplicating this code from the above? Could `_concat` instead be modified to handle the no-output case?

Reply:
Refactored in a function the hashing to be reused. :ok_hand: